include/boost/geometry/algorithms/detail/direction_code.hpp

```diff
@@ -140,12 +157,12 @@ struct direction_code_impl<spherical_equatorial_tag>
         calc_t const dlat1 = latitude_distance_signed<units_t, calc_t>(b1, a1, dlon1, is_antilon1);
         calc_t const dlat2 = latitude_distance_signed<units_t, calc_t>(b1, p1, dlon2, is_antilon2);
 
-        calc_t mx = is_a_pole || is_b_pole || is_p_pole ?
-                    c0 :
-                    (std::min)(is_antilon1 ? c0 : math::abs(dlon1),
-                               is_antilon2 ? c0 : math::abs(dlon2));
-        calc_t my = (std::min)(math::abs(dlat1),
-                               math::abs(dlat2));
+        calc_t const mx = is_a_pole || is_b_pole || is_p_pole ?
+                            c0 :
+                            (std::min)(is_antilon1 ? c0 : math::abs(dlon1),
+                                    is_antilon2 ? c0 : math::abs(dlon2));
```

Comment:
should this be aligned with `is_antilon1` in above line?

Reply:
Indeed. :heavy_check_mark: 
Updated (I usually align in another way, changed to that)